code/modules/atmospherics/machinery/atmospherics.dm

```diff
@@ -190,27 +190,31 @@ Pipelines + Other Objects -> Pipe network
 		var/E = env_air ? env_air.return_pressure() : 0
 		var/internal_pressure = I - E
 
-		playsound(src.loc, W.usesound, 50, 1)
+		playsound(loc, W.usesound, 50, 1)
 		to_chat(user, "<span class='notice'>You begin to unfasten \the [src]...</span>")
+
+		for(var/obj/item/clothing/shoes/magboots/usermagboots in user.get_equipped_items())
+			if(usermagboots.gustprotection && usermagboots.magpulse)
+				safefromgusts = TRUE
+
 		if(internal_pressure > 2*ONE_ATMOSPHERE)
-			to_chat(user, "<span class='warning'>As you begin unwrenching \the [src] a gust of air blows in your face... maybe you should reconsider?</span>")
 			unsafe_wrenching = TRUE //Oh dear oh dear
+			if(internal_pressure > 1750 && !safefromgusts) // 1750 is the pressure limit to do 60 damage when thrown
+				to_chat(user, "<span class='userdanger'>As you struggle to unwrench \the [src] a huge gust of gas blows in your face! This seems like a terrible idea!</span>")// give them more time to decide this is a bad idea, since it might kill them
```

Comment:
this doesnt actually give them more time anymore.

Reply:
Qwerty thought the slowdown should be removed, do you think it should maybe be something like 1.5 or 2? because I can add it back.

Reply:
oh the comment, my bad.